docs/REST API/rest-api.md

```diff
@@ -2257,6 +2262,7 @@ Returns the information of the specified host.
         "name": "osquery",
         "version": "4.5.1",
         "source": "rpm_packages",
+        "browser": "",
```

Comment:
@rachaelshaw do we return `browser` for every software item.

I would expect to see it for browser extensions only.

Reply:
Confirmed that we do return `browser` for every software item.

Filed a bug to fix this here: https://github.com/fleetdm/fleet/issues/16056